cli.js

```diff
@@ -1,8 +1,39 @@
-import { main, log } from "./app.js";
+import {
+  validatePostcode,
+  geocodePostcode,
+  getCoordinates,
+  log,
+} from "./postcodes.js";
 
-let base_maps_url = "http://maps.google.com/maps?z=12&t=m&q=loc:";
+const base_maps_url = "http://maps.google.com/maps?z=12&t=m&q=loc:";
 
-let coordinates = await main();
+export function getPostcodeFromArgs() {
+  let postcode = process.argv[2];
```

Comment:
```suggestion
  const postcode = process.argv[2];
```
Could this also be a `const`? if only for consistency with other bits?